### ISSUE TITLE
Raise Not Implemented - Launching run_classroom.py

### ISSUE THREAD

**Issue by Tlospock** [petrosgk/MalmoRL#2]
Hello,

I'm trying to get this MalmoRL framework to run, to try to implement personal missions.
But I can't even get the examples included to run.

The error I keep getting is the following:
![image](https://user-images.githubusercontent.com/16708864/38226797-53afe574-3736-11e8-9c0b-504124ee42ad.png)

This is raised because the compile function is empty, excepted for the exception raising.
`
def compile(self, optimizer, metrics=[]):
        """Compiles an agent and the underlaying models to be used for training and testing.
        # Arguments
            optimizer (`keras.optimizers.Optimizer` instance): The optimizer to be used during training.
            metrics (list of functions `lambda y_true, y_pred: metric`): The metrics to run during training.
        """
raise `NotImplementedError()
`

I tried getting the keras-rl module from pip, and from the git fork.

Is thi normal? Do you have a way to get an example up and running?

**Comment by petrosgk** [comment]
Yeah sorry, that's caused because I updated the `RandomAgent` in keras-rl to not require the call to `compile()` as it didn't make sense but forgot to remove it from MalmoRL. It should be fixed now.
